README.md

```diff
@@ -24,7 +26,7 @@ To use composer:
     ```json
     {
         "require": {
-            "hellosign/hellosign-php-sdk": "3.*@dev"
+            "hellosign/hellosign-php-sdk": "3.4.6"
```

Comment:
I worry about this line actually being kept up to date; I think a semantic version constraint would be the better way to go here: `"^3.0"`

(This will tell composer to grab the latest stable release before 4.x. I just tested to make sure.)